tests/Test/ConfigEventsTest.php

```diff
@@ -87,14 +97,18 @@ public function testLoadEmptyFixturesAndCheckEventsAreCalled(string $eventName,
         ;
 
         // Register to the event
-        $eventDispatcher = $this->getContainer()->get('event_dispatcher');
+        $eventDispatcher = self::$container->get('event_dispatcher');
         $eventDispatcher->addListener(
             $eventName,
             [$mock, $methodName]
         );
 
         // By loading fixtures, the events will be called (or not)
-        $fixtures = $this->loadFixtures([]);
+        if ($withCache) {
```

Comment:
just FYI .. in such an if/else situation with single line assignments to the same variable, I personally prefer a ternary for readability. but I have no intention of enforcing such a personal preference